content/tugboat-cli/install-the-cli.md

```diff
@@ -15,6 +15,7 @@ binary.
 - [Windows](#windows)
 - [MacOS](#macos)
 - [Linux](#linux)
+- [Homebrew](#homebrew)
```

Comment:
Since we only support homebrew for MacOS, maybe this should be bundled in the mac section? In fact, I'd go so far as to say that homebrew is our recommended installation method, and we can just remove the manual steps.

Reply:
Happy to move it to MacOS section & make it recommended, but I'd like to leave in the manual steps for folks who don't want to use Homebrew. I know some developers who hate brew and would rather manage things manually. (I don't understand it... but they exist.)